akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -423,6 +423,18 @@ object ShardRegion {
     }
   }
 
+  /**
+   * Discover if the shard region is registered with the coordinator.
+   * Not serializable as only to be sent to the local shard region
+   * Response is [[ShardRegionState]]
+   */
+  final object GetShardRegionStatus extends ShardRegionQuery
```

Comment:
Start out as internal API and open up if requested?

Reply:
could we expand `CurrentShardRegionState` instead?

Reply:
That does an ask to every shard so a bit overkill

Reply:
ok

Reply:
Making this internal for now